docs/controls/TextToolbar.md

```diff
@@ -39,7 +39,7 @@ You can also Remove Default Buttons from any format, or Add Custom buttons to an
 | Editor | RichEditBox | Gets or sets the RichEditBox to Attach to, this is required for any formatting to work |
 | Format | [Format](/dotnet/api/microsoft.toolkit.uwp.ui.controls.texttoolbarformats.format) | Gets or sets which formatter to use, and which buttons to provide |
 | Formatter | [Formatter](/dotnet/api/microsoft.toolkit.uwp.ui.controls.texttoolbarformats.formatter) | Gets or sets the formatter which is used to format the text from the buttons |
-| Labels | [TextToolbarStrings](/dotnet/api/microsoft.toolkit.uwp.ui.controls.texttoolbarstrings) | Gets or sets the default string Labels |
+| Labels | [TextToolbarStrings](https://www.fuget.org/packages/Microsoft.Toolkit.Uwp.UI.Controls/5.0.0/lib/uap10.0.16299/Microsoft.Toolkit.Uwp.UI.Controls.dll/Microsoft.Toolkit.Uwp.UI.Controls/TextToolbarStrings) | Gets or sets the default string Labels |
```

Comment:
Is pointing to fuget the most appropriate URL we have for this?

Reply:
It seemed like a good replacement, lacking other options.